dom.bs

```diff
@@ -5769,6 +5772,33 @@ This and <a lt="other applicable specifications">other specifications</a> may de
 <a for=/>elements</a>. The algorithm is passed <var>element</var>, <var>localName</var>,
 <var>oldValue</var>, <var>value</var>, and <var>namespace</var>.
 
+<p>The [=insertion steps=] for an {{Element}} (<var>inserted</var>) are as follows:
+
+<ol>
+ <li>
+  <p>If <var>inserted</var> is [=connected=], and it has a content attribute (<var>attr</var>) whose
```

Comment:
It should indeed. Fixed in https://github.com/whatwg/dom/pull/436/commits/3e4bffb3b35516c3fdfe55f4965bc869500208db.